packages/clarity-js/src/core/index.ts

```diff
@@ -41,7 +41,8 @@ export function check(): boolean {
             document["createTreeWalker"] &&
             "now" in Date &&
             "now" in performance &&
-            typeof WeakMap !== "undefined";
+            typeof WeakMap !== "undefined" &&
+            navigator['globalPrivacyControl'] !== true
```

Comment:
can you use (navigator && "(navigator && "globalPrivacyControl" in navigator") before testing for true - that is similar to how we extract navigator values in other places